Libraries/Opc.Ua.Configuration/ApplicationInstance.cs

```diff
@@ -294,18 +312,8 @@ public async Task<ApplicationConfiguration> LoadApplicationConfiguration(string
         public async Task<ApplicationConfiguration> LoadApplicationConfiguration(bool silent)
         {
             string filePath = ApplicationConfiguration.GetFilePathFromAppConfig(ConfigSectionName);
-            ApplicationConfiguration configuration = await LoadAppConfig(
-                silent, filePath, ApplicationType, ConfigurationType, true, CertificatePasswordProvider)
-                .ConfigureAwait(false);
-
-            if (configuration == null)
-            {
-                throw ServiceResultException.Create(StatusCodes.BadConfigurationError, "Could not load configuration file.");
-            }
-
-            m_applicationConfiguration = FixupAppConfig(configuration);
 
-            return m_applicationConfiguration;
+            return await LoadApplicationConfiguration(filePath, silent);
```

Comment:
nitpick: remove await and async keyword in function header or
add ConfigureAwait(false). Better ist the former...